src/translations/fr.json

```diff
@@ -208,6 +210,7 @@
   "invalidFormatOrName": "Format ou nom du fichier importé non valide",
   "parameterLoadingProblem": "Problème de chargement des paramètres",
   "dragAndDrop": "glisser déposer",
+  "changeOperatorMessage": "Voulez-vous changer l'opérateur ? Le nouvel opérateur s'appliquera sur toutes les règles déjà créées dans le groupe.",
```

Comment:
crées

Reply:
No, it's correct (see https://www.reverso.net/orthographe/correcteur-francais/#text=Le%2520nouvel%2520op%25C3%25A9rateur%2520s'appliquera%2520sur%2520toutes%2520les%2520r%25C3%25A8gles%2520d%25C3%25A9j%25C3%25A0%2520cr%25C3%25A9es%2520dans%2520le%2520groupe)

Reply:
OK. I don't know that but it is interesting the AI tool..